wordpress-mobile/gutenberg-mobile
# Refactor icon customization in block picker

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1004

gutenberg PR https://github.com/WordPress/gutenberg/pull/15778

To test:

Display the block picker and make sure all icons have the same color: `$gray-dark` and same size: `32px`

<img width="336" alt="Screen Shot 2019-05-22 at 19 42 53" src="https://user-images.githubusercontent.com/5032900/58192610-d1b3c100-7cc9-11e9-9c8b-490b816dca50.png">

Update release notes:
